public/resources/js/contributers-wall.js

```diff
@@ -0,0 +1,79 @@
+async function getContributors() {
+    const cacheKey = 'vrc-get/contributors';
+    const cachedResult = JSON.parse(localStorage.getItem(cacheKey));
+
+    // Check if cached result exists and is not expired
+    if (cachedResult && Date.now() - cachedResult.timestamp < 1800000) { // 30 minutes = 1800 seconds = 1,800,000 milliseconds
+        console.info('Got contributers from cache instead of API.')
+        return cachedResult.data;
+    }
+
+    // If not cached or expired, fetch from GitHub
+    try {
+        const response = await fetch(`https://api.github.com/repos/vrc-get/vrc-get/contributors`);
```

Comment:
I think we should avoid implicitly use viewer's github api quota so I think it's better to use github api on build.
I think contributors ranking is not changed frequently so I think it's ok to update on release or other website updates.

(this is one of reasons why ALCOM website is using updater json for showing latest version)

Reply:
Actually, the reason why we use tauri api json for ALCOM release is to not create build process for website so I think it might be good idea to embed version info with handlebars